### DISCUSSION TITLE
Pauses

### DISCUSSION THREAD

**Discussion by ramonesg** [flybywiresim/aircraft#2315] [Help]
Does anyone knows what can be causing this? I’m getting some stutters with occasional pauses but then it goes back to smooth experience

ive removed everything from the community folder

I don’t know how to post a video

**Comment by nyhmbo** [comment]
so it happens even without the mod installed? check if your graphics drivers are up to date. might be a problem with that.
you could upload a video somewhere else and post it here.

**Reply by ramonesg** [reply]
I haven’t tried without the mod! Drivers are up to day! I’ll try to upload the video

**Reply by nyhmbo** [reply]
okay. also make sure you are on either the latest stable or dev version of the mod

**Reply by ramonesg** [reply]
> so it happens even without the mod installed? check if your graphics drivers are up to date. might be a problem with that.
> you could upload a video somewhere else and post it here.

https://streamable.com/htbjcr

**Reply by nyhmbo** [reply]
I would try without the mod or doing a fresh install of the mod. low fps could be alot of stuff. doesn't need to be a mod issue.

**Reply by ramonesg** [reply]
Looks like without the mod it doesn’t happens so should I remove the mod from the community and do a clean install?

**Reply by nyhmbo** [reply]
yea, doesn't hurt.

**Comment by MapleNAV-TCASFAULT** [comment]
It would be great to know what your PC specs are too! If its stuttering really bad tune down your LOD on object and terrain as it may be your CPU taking its time to load everything.

**Reply by ramonesg** [reply]
> It would be great to know what your PC specs are too! If its stuttering really bad tune down your LOD on object and terrain as it may be your CPU taking its time to load everything.

Installing the mod again didn’t help but now it’s doing it even on default!

Specs 
I7-9700k 5.1
32gb ram
Gtx 2080

**Reply by MapleNAV-TCASFAULT** [reply]
Does it happen everywhere or at selected airports?

**Reply by ramonesg** [reply]
I believe in some airports I will check,
For now I’ve reduced the LOD OBJ AND TERR as you mentioned to 120-120 it was at 200

**Comment by NathanInnes** [comment]
If you go in the sim and enable dev tools then show FPS counter, it may give us some more insight on what might be causing it to stutter.

**Reply by ramonesg** [reply]
What exactly should y look at?

**Reply by MapleNAV-TCASFAULT** [reply]
If its limited by GPU, you have a GPU bottleneck. If its limited by mainthread, you have a CPU bottleneck.

**Reply by ramonesg** [reply]
It says both , keep changing but more GPU than CPU but I’ve never had this issue before it was after the world usa patch
